crates/router/src/core/payment_methods/utils.rs

```diff
@@ -0,0 +1,857 @@
+                                .into_iter()
+                                .map(|country| common_enums::Country::from_alpha2(*country))
+                                .collect();
+                        let dir_countries: Vec<dir::DirValue> = config_countries
+                            .into_iter()
+                            .map(dir::DirValue::BillingCountry)
+                            .collect();
+
+                        let config_country_agg_node = builder
+                            .make_in_aggregator(dir_countries, None, None::<()>)
+                            .map_err(KgraphError::GraphConstructionError)?;
+
+                        let node = builder
+                            .make_all_aggregator(
+                                &[
+                                    (
+                                        pm_object_country_value_node,
+                                        cgraph::Relation::Positive,
+                                        cgraph::Strength::Strong,
+                                    ),
+                                    (
+                                        config_country_agg_node,
+                                        cgraph::Relation::Positive,
+                                        cgraph::Strength::Strong,
+                                    ),
+                                ],
+                                None,
+                                None::<()>,
+                                None,
+                            )
+                            .map_err(KgraphError::GraphConstructionError)?;
+                        return Ok(Some(node));
+                    }
+                }
+            }
+        }
+        admin::AcceptedCountries::DisableOnly(countries) => {
+            if let Some(config) = config
+                .0
+                .get(connector.as_str())
+                .or_else(|| config.0.get("default"))
+            {
+                if let Some(value) =
+                    config
+                        .0
+                        .get(&settings::PaymentMethodFilterKey::PaymentMethodType(
+                            *payment_method_type,
+                        ))
+                {
+                    // Country from the MCA
+                    let pm_object_country_value_node = builder
+                        .make_in_aggregator(
+                            countries
+                                .into_iter()
+                                .map(|country| {
+                                    dir::DirValue::BillingCountry(
+                                        common_enums::Country::from_alpha2(country),
+                                    )
+                                })
+                                .collect(),
+                            None,
+                            None::<()>,
+                        )
+                        .map_err(KgraphError::GraphConstructionError)?;
```

Comment:
same here. This should probably be created regardless of whether the config exists or not.